proposals/flexible-vectors/FlexibleVectors.md

```diff
@@ -26,10 +28,12 @@ size of the lane in bits, `S.Lanes` is the number of lanes, which is dynamic.
 
 |    S      | S.LaneBits |
 |-----------|-----------:|
-| `vec.v8`  |          8 |
-| `vec.v16` |         16 |
-| `vec.v32` |         32 |
-| `vec.v64` |         64 |
+| `vec.i8`  |          8 |
+| `vec.i16` |         16 |
+| `vec.i32` |         32 |
+| `vec.f32` |         32 |
+| `vec.i64` |         64 |
+| `vec.f64` |         64 |
```

Comment:
Unsigned types seem missing. Is it on purpose?

Reply:
Yep, the idea was not to end up with an integer value used in place of a floating-point value and vice versa. Scalar Wasm does not protect against signed-unsigned mismatch, so I did not think it was worth to do that here.